Modules/AppointmentSlots/Services/AppointmentSlotsService.php

```diff
@@ -212,8 +213,16 @@ private function getUserFreeSlots($userId)
             return $date->format(config('constants.date_format', 'Y-m-d'));
         });
 
-        $datesToRemove = $reservedSlotsCount->filter(function ($value, $key) {
-            return $value >= config('hr.daily-appointment-slots.max-reserved-allowed', 3);
+        $datesToRemove = $reservedSlotsCount->filter(function ($value, $key) use ($userId) {
+            $userMeta = User::find($userId)->meta;
```

Comment:
```suggestion
            $userMeta = UserMeta::where('user_id', $user_id)->first();
```

Will there be any difference in end result if I do this?

Reply:
I have tried using where clause comparing user_id with $userId but no slots were visible for that particular user although I had fixed it as 1.